nearcore/src/state_sync.rs

```diff
@@ -139,6 +143,47 @@ impl StateSyncDumpHandle {
     }
 }
 
+/// Fetches the state sync header from DB and serializes it.
+fn get_serialized_header(
+    shard_id: ShardId,
+    sync_hash: CryptoHash,
+    chain: &Chain,
+) -> anyhow::Result<Vec<u8>> {
+    let header = chain.get_state_response_header(shard_id, sync_hash)?;
+    let mut buffer: Vec<u8> = Vec::new();
+    header.serialize(&mut buffer)?;
+    Ok(buffer)
+}
+
+/// Check if the state sync header exists in the external storage.
+async fn is_state_sync_header_missing_for_epoch(
```

Comment:
nit: It's typically easier to work with positive statements such as "exists" than negative such as "missing". Also it would be more compatible with the comment.